Mathlib/Geometry/Euclidean/Angle/Unoriented/Basic.lean

```diff
@@ -5,6 +5,9 @@ Authors: Joseph Myers, Manuel Candales
 -/
 import Mathlib.Analysis.InnerProductSpace.Subspace
 import Mathlib.Analysis.SpecialFunctions.Trigonometric.Inverse
+import Mathlib.LinearAlgebra.CrossProduct
+import Mathlib.Analysis.InnerProductSpace.PiL2
+
```

Comment:
Typically we try to keep the imports alphabetical. Also, you seem to have a stray newline here and at the bottom of the file.

Reply:
fixed!